src/test/suite/fixtures/recorded/languages/rust/chuckArgBlueEight.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
+command:
+  spokenForm: chuck arg blue eight
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: blue, character: '8'}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    enum IpAddr {
+        V4(u8, u8, u8, u8),
+        V6(String),
+    }
+  selections:
+    - anchor: {line: 4, character: 0}
+      active: {line: 4, character: 0}
+  marks:
+    blue.8:
+      start: {line: 1, character: 19}
+      end: {line: 1, character: 21}
+finalState:
+  documentContents: |
+    enum IpAddr {
+        V4(u8, u8, u8),
```

Comment:
The problem with these tests where they're all `u8` is that it's difficult to tell which one got deleted.  Could you possibly do the tests using different types, eg `Foo(u8, u16, u32)`?  Having examples where the code is "meaningful" is a non-goal in my mind.  We're just testing syntactic structures